hedera-node/docs/design/services/smart-contract-service/update_token_custom_fees_sc.md

```diff
@@ -0,0 +1,87 @@
+# Update Token Custom Fees via Smart Contracts
+
+## Purpose
+
+In order to support updating the Token Custom Fees via smart contract calls, we need to expose new smart contract functions in the Hedera Smart Contract Service (HSCS) logic exposing Hedera Token Service (HTS) logic. This document will define the architecture and implementation of the `updateFungibleTokenCustomFees` and `updateNonFungibleTokenCustomFees` smart contract functions that will extend tha capabilities of the HSCS.
+
+## References
+
+[HIP-18](https://hips.hedera.com/hip/hip-18) - HIP that introduces token custom fees.\
```

Comment:
funky trailing `\` here and on next line?

Reply:
It's shorter way to add a line break.

Reply:
IDNKT.   Looked it up and apparently it works on some markdown parsers (e.g., github and intellij) but not all.  And it's backwards if you ask me since normally in programming languages a backslash at the end of the line _removes_ the line break.  But ok, I guess.
* The "official" markdown solutions are terrible:  a) `<br>` at end of line: ugly; b) two space characters at the end of a line which first, are typically invisible, and second, most people have their editors set to strip trailing blanks.

Or you could make this a bulleted list if you want.